client/flaschen.py

```diff
@@ -0,0 +1,64 @@
+# -*- mode: c++; c-basic-offset: 4; indent-tabs-mode: nil; -*-
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://gnu.org/licenses/gpl-2.0.txt>
+
+import socket
+
+class Flaschen(object):
+  '''A Framebuffer display interface that sends a frame via UDP.'''
+
+  def __init__(self, host, port, width, height, layer=0):
+    self.host = host
+    self.port = port
+    self.width = width
+    self.height = height
+    self.layer = layer
+    self.pixels = []
+    for x in xrange(width):
+      self.pixels.append([(0, 0, 0) for y in xrange(height)])
+    self.sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+
+  def _header(self):
+    return ''.join(["P6\n",
+                    "45 35\n",
```

Comment:
This is using the hardcoded 45, 35 values instead of self.width, self.height.

Also: It probably makes sense to pre-calculate the header and footer values instead of doing a string-join() every single time.